gulpfile.js

```diff
@@ -65,6 +65,22 @@ function test() {
 gulp.task('build-test', ['build'], test);
 gulp.task('test', test);
 
+gulp.task('checkImports', function (cb) {
+    var checkProcess = child_process.fork(path.join(__dirname, "tools", "checkCasing.js"),
+        {
+            cwd: path.resolve(__dirname, "src"),
+            stdio: "inherit"
+        });
+    checkProcess.on("error", cb);
```

Comment:
These two lines may call cb multiple times according to the docs. Is that okay in this case?